.rhcicd/grafana/grafana-dashboard-notifications-general.configmap.yaml

```diff
@@ -2369,7 +2547,7 @@ data:
       "timezone": "",
       "title": "Notifications Dashboard",
       "uid": "KQIVyFuMk",
-      "version": 10
+      "version": 1
```

Comment:
updating the version to 11 would be great. Then we would be able to restore the older dashboards. Not sure if we would ever need it, but it could be useful.

Reply:
Ok I thought this was ignored and rolling back in this repo would be enough. Will send an update